cleverhans/utils_pytorch.py

```diff
@@ -79,5 +79,13 @@ def tf_model_fn(x_op):
                                      grad_func=_tf_gradient_fn)[0]
         out.set_shape([None, out_dims])
         return out
+      
+    def _tf_fprop_fn(x_np):
```

Comment:
It looks like this new function that you're adding is never used and is unreachable after definition. Am I missing something @cryptoannavan ?